bower.json

```diff
@@ -29,9 +29,9 @@
     "purescript-integers": "^6.0.0",
     "purescript-js-date": "^8.0.0",
     "purescript-maybe": "^6.0.0",
-    "purescript-node-buffer": "^8.0.0",
+    "purescript-node-buffer": "^9.0.0",
     "purescript-node-path": "^5.0.0",
-    "purescript-node-streams": "^7.0.0",
+    "purescript-node-streams": "#684041e14e56c75c0bf49db0e556aec6d0248e5a",
```

Comment:
Note: this hash is only being used so development on this repo can start. This needs to be updated to an actual release before this repo can be released.